src/pages/cheoljinJu/Main/Feed/Feed.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import Comment from '../Comment/Comment';
+import CommentForm from '../CommentForm/CommentForm';
+import './Feed.scss';
+
+class Feed extends Component {
+  state = {};
+
+  componentDidMount = () => {
+    this.setState(this.props.feed);
+  };
+
+  handleAdd = value => {
+    const comment = {
+      id: Date.now(),
+      userName: '24_Wecode',
+      comment: value,
+      isUser: true,
+      isLike: false,
+    };
+    const comments = [...this.state.comments, comment];
+    this.setState({ comments });
+  };
+
+  handleDelete = reply => {
+    const comments = this.state.comments.filter(item => reply.id !== item.id);
+    this.setState({ comments });
+  };
+
+  handleLike = reply => {
+    const comments = this.state.comments.map(item => {
+      if (item.id === reply.id) {
+        item.isLike = !item.isLike;
+        return item;
+      } else {
+        return item;
+      }
+    });
+    this.setState({ comments });
+  };
```

Comment:
- comments -> newComments 로 이름 수정해 주세요! 기존의 comments 와는 다른 새로운 댓글 배열임을 명시해주시는 게 좋습니다.
- 여기서 item 은 `원본 요소` 입니다. 원본 요소의 isLike 프로퍼티에 직접 반대 값을 할당하고 있습니다. 즉, state 를 직접 수정하고 계신데, 원본은 건드리지 말고 새로운 객체를 만들어서 업데이트 하도록 코드 수정해 주세요!
```suggestion
    const newComments = this.state.comments.map(item => {
      if (item.id === reply.id) {
        return { ...item, isLike: !item.isLike }
      } else {
        return item;
      }
    });
    this.setState({ newComments });
  };
```

Reply:
성훈님 `map()` 메소드의 경우에 새로운 배열을 반환하는 것으로 알고 있어서 각 `item.isLike`를 변경했었는데, 
말씀해주신 것 보고 생각해보니 배열 안에 객체 형태로 있기 때문에 직접 형태를 변경하고 있음을 알게 되었습니다. 감사합니다! 
수정하고 참조 형태의 데이터를 사용할 때 얕은 복사와 깊은 복사 등에 대해서도 더 공부해보겠습니다!

Reply:
`setState({comments})` 부분은 동일한 이름의 변수를 함수 내에서 선언하고 사용하면 생략이 가능해서 편리하다고 느꼈었는데 
`setState({comments : newComments})` 의 형식으로 작성해주는 것이 유지보수나 가독성을 위해서 좋다고 생각하면 될까요?